packages/palette/src/elements/MultiSelect/MultiSelect.tsx

```diff
@@ -43,6 +47,7 @@ export const MultiSelect: React.FC<MultiSelectProps> = ({
 }) => {
   const [visible, setVisible] = useState(false)
   const [selection, setSelection] = useState<Option[]>([])
+  const [boxProps] = splitBoxProps(rest)
```

Comment:
The use of `splitBoxProps` means you are omitting some other props here, which would be the second argument. I don't think you need this and can just spread rest props on the outer box.

Reply:
I just pushed a commit that removes the const. I think this can be resolved, but let me know!